packages/docs/getting-started/running-a-validator.md

```diff
@@ -331,6 +342,8 @@ celocli lockedgold:lock --from $CELO_VALIDATOR_GROUP_ADDRESS --value 10000000000
 celocli lockedgold:lock --from $CELO_VALIDATOR_ADDRESS --value 10000000000000000000000
 ```
 
+Since your balance is in fact higher than this, you may wish to lock more with these accounts.
```

Comment:
"This amount (10,000 Celo Gold) represents the minimum amount needed to be locked in order to register a Validator and Validator group. Since your balance is in fact higher than this, you may wish to lock more with these accounts. Note that you will want to be sure to leave enough Gold unlocked to be able to continue to pay transaction fees for future transactions." 

"Note that the CLI commands pay transaction fees in Celo Gold. Be sure not to lock up all of your Celo Gold,